rdf-star-cg-spec.html

```diff
@@ -173,6 +173,30 @@ <h2>Concepts and Abstract Syntax</h2>
     </ul>
 
     <p>Again, this definition is an extension of the notion of <dfn data-cite="RDF11-CONCEPTS#dfn-rdf-dataset">RDF dataset</dfn>, hence it follows that any <a>RDF dataset</a> is also an <a>RDF* dataset</a>.</p>
+
+    <section class="informative" id="type-token">
+      <h2>RDF* triples and the type-token distinction</h2>
+
+      <p>According to the definitions above, an <a>RDF* triple</a> is an abstract entity whose identity is entirely defined by its subject, predicate an object. In other words, given three <a>RDF* terms</a> |s|, |p| and |o|, the embedded triple (|s|, |p|, |o|) is assumed to represent the <em>same thing</em> everywhere it occurs, including across different graphs (just like the same IRI |p| is assumed to represent the same thing everywhere it occurs).</p>
+
+      <p>In some situations, however, it might be necessary to distinguish the <em>occurrences</em> of a triple (in different files, for example). Consider the following sentence: "The triple `:s :p :o` was added in file1.ttl by Alice, and the same triple was added in file2.ttl by Bob." Note that words "same triple" in this sentence may be confusing, because although the triple (as an abstract entity) is the same, its respective occurrences in the two files are different things, each within a different file and with a different author. This is known as the <a href="https://en.wikipedia.org/wiki/Type%E2%80%93token_distinction">type-token distinction</a>. As the embedded triple represents a unique abstract entity, adequately conveying the meaning of the sentence above requires additional nodes to represent each occurrence, as illustrated below.</p>
```

Comment:
```suggestion
      <p>In some situations, however, it might be necessary to distinguish the <em>occurrences</em> of a triple (in different Turtle or Turtle* files, for example). Consider the following sentence: "The triple `:s :p :o` was added in file1.ttl by Alice, and the same triple was added in file2.ttl by Bob." Note that words "same triple" in this sentence may be confusing, because although the triple (as an abstract entity) is the same, its respective occurrences in the two files are different things, each within a different file and with a different author. This is known as the <a href="https://en.wikipedia.org/wiki/Type%E2%80%93token_distinction">type-token distinction</a>. As the embedded triple represents a unique abstract entity, adequately conveying the meaning of the sentence above requires additional nodes to represent each occurrence, as illustrated below.</p>
```